src/app/app.component.scss

```diff
@@ -1,19 +1,10 @@
 @import '~@pxblue/colors/palette.scss';
 
-::ng-deep .pxb-blue .mat-drawer-container {
-    background-color: map-get($pxb-white, 50) !important;
-    min-height: 100vh !important;
-}
-
-.right-panel {
-    height: 100vh;
-    border-left: 1px solid map-get($pxb-black, 50);
-    padding: 1rem;
-    width: 270px;
+html,
+body {
+    height: 100%;
 }
 
-@media screen and (max-width: 960px) {
-    .right-panel {
-        display: none;
-    }
+.pxb-drawer-layout-nav-content {
+    background-color: map-get($pxb-white, 50) !important;
 }
```

Comment:
these can go in styles.scss since they are global rules (and you won't need the important). 
styles.scss should also remove the import of the theme...that's handled in angular.json